gen_sig.js

```diff
@@ -0,0 +1,14 @@
+const secretKey = '9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08';
```

Comment:
Hard-coding the secret key into the file allows anyone to forge signatures immediately.  This file should take the secret key as an input from a file, then the specific file path can be specified as a comand-line input.

Reply:
The signature process should be done when the JSON is created (so I believe in the challenge-bypass-extension repository). I created this file to give an example of how to generate the signature given the new JSON structure. The files `gen_sig` and `ver_sig` are not meant for production, but will help me explain my ideas to the interviewer.